stub/src/demangle.cpp

```diff
@@ -16,24 +16,45 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-#include <cstdlib>
-#if !defined(_MSC_VER)
+#ifndef _MSC_VER
 #include <cxxabi.h>
-#endif
+#include <memory>
+#endif // !_MSC_VER
 
 #include "neurala/utils/demangle.h"
 
 namespace neurala
 {
-std::unique_ptr<char, void (*)(void*)>
-demangle(const char* name) noexcept
+std::string
+demangle(std::string_view name)
```

Comment:
I just realized that using `string_view` forces c++17 on the users - is this an issue? should I return `const char*`

Reply:
B4B uses C++17 forcing a match will avoid potential ABI issues.